polaris-react/stylelint/plugins/coverage/configs/index.js

```diff
@@ -0,0 +1,127 @@
+module.exports = {
+  plugins: ['stylelint-scss', '../'],
+  rules: {
+    'at-rule-disallowed-list': ['keyframes'],
+    'color-named': 'never',
+    'color-no-hex': true,
+    'declaration-property-value-disallowed-list': [
+      {
+        '/color$/': ['/#[0-9a-f]{3,6}/'],
+        display: ['grid', 'flex'],
+        'font-weight': ['/[0-9]*/'],
+        opacity: ['/?![01]'],
+        'z-index': ['/[0-9]*/'],
+      },
+    ],
+    'function-disallowed-list': [
+      'brightness',
+      'contrast',
+      'hue-rotate',
+      'hsl',
+      'hsla',
+      'invert',
+      'rgb',
+      'rgba',
+      'sepia',
+    ],
+    'property-disallowed-list': [
+      'position',
+      'grid',
+      'flex',
+      'grid-row',
+      'grid-row-start',
+      'grid-row-end',
+      'grid-column',
+      'grid-column-start',
+      'grid-column-end',
+      'grid-template',
+      'grid-template-areas',
+      'grid-template-rows',
+      'grid-template-columns',
+      'grid-area',
+    ],
+    'unit-disallowed-list': ['px', 'rem', 'em', 's', 'ms'],
```

Comment:
Shouldn't we allow `px`s, `rem`s, and `em`s? `polaris-react` has a `postcss` plugin to convert `px-to-rems` and I believe we use `em`s for all our media queries.

When we move this over to `@shopify/stylelint-polaris`, this might be one of those rules where we have different configs for internal `polaris-react` usage and Polaris consumers